packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.service.js

```diff
@@ -295,6 +295,16 @@ export default class PciProjectStorageBlockService {
       .then(({ data }) => data);
   }
 
+  get3azAvailability(projectId) {
+    return this.getCatalog(projectId).then((catalog) => {
+      return catalog.filters.deployment.some((d) => d.name === 'region-3-az');
```

Comment:
```suggestion
      return catalog.filters.deployment.some((d) => d.name === $ctrl.MY_SUPER_CONSTANT);
```
maybe if possible move 'region-3-az' in constants file